BGL/include/CGAL/boost/graph/iterator.h

```diff
@@ -161,7 +161,26 @@ struct Face {
     return face(h,*g);
   }
 };
+template <typename G>
+struct OppositeFace {
```

Comment:
Changed, merged in integration and pushed to mine

andreas

On 01/06/2015 12:10, Laurent Rineau wrote:
> In BGL/include/CGAL/boost/graph/iterator.h
> <https://github.com/CGAL/cgal/pull/99#discussion_r31413496>:
>
>> @@ -161,7 +161,26 @@ struct Face {
>>      return face(h,*g);
>>    }
>>  };
>> +template <typename G>
>> +struct OppositeFace {
>
> Browsing more in the file, I discover that the whole file is written
> that way. What is the rational? The naming convention you have followed
> looks like the one we have for concepts in the CGAL documentation.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/CGAL/cgal/pull/99/files#r31413496>.
>